security_monkey/watchers/config_recorder.py

```diff
@@ -20,20 +20,32 @@
 
 
 """
-from security_monkey.decorators import record_exception, iter_account_region
+
 from security_monkey.watcher import Watcher
 from security_monkey.watcher import ChangeItem
+from security_monkey.constants import TROUBLE_REGIONS
+from security_monkey.exceptions import BotoConnectionIssue
 from security_monkey import app
+from boto.ec2 import regions
+from security_monkey.decorators import record_exception, iter_account_region
 
 
 class ConfigRecorder(Watcher):
     index = 'configrecorder'
     i_am_singular = 'Config Recorder'
     i_am_plural = 'Config Recorders'
 
+    # TODO: Replace hardcoded region with `get_available_regions` after next boto3 release
+    # Specific PR here: https://github.com/boto/boto3/pull/531/files
+    # AWS Config currently only supports the following regions
+    regions = ["us-east-1", "us-west-1", "us-west-2", "eu-west-1", "eu-central-1",
```

Comment:
@monkeysecurity we have the correct code internally and it looks like it is correct in you master. We will push a fix for this branch.